python/ray/train/trainer.py

```diff
@@ -172,12 +181,11 @@ def _get_backend_config(
         if isinstance(backend, BackendConfig):
             return backend
         elif isinstance(backend, str):
-            try:
-                return BACKEND_NAME_TO_CONFIG_CLS[backend]()
-            except KeyError:
+            if backend not in BACKEND_NAME_TO_CONFIG_CLS_NAME:
                 raise ValueError(f"Invalid backend: {backend}. "
                                  f"Supported string values are: "
-                                 f"{BACKEND_NAME_TO_CONFIG_CLS.keys()}")
+                                 f"{BACKEND_NAME_TO_CONFIG_CLS_NAME.keys()}")
```

Comment:
nit: Move this into `get_backend_config_cls` - keeps all logic around `BACKEND_NAME_TO_CONFIG_CLS_NAME` in 1 method.

Reply:
Done!